Makefile

```diff
@@ -95,3 +97,31 @@ check-inside-toolbox:
 
 requre-purge-files:
 	pre-commit run requre-purge --all-files --verbose --hook-stage manual
+
+# run all the pods needed by the service pod
+# use docker-compose to update and run them all
+# stop the service pod to be able to re-run it
+# later with different options
+compose-for-migrate-db:
+	$(COMPOSE) up --build --force-recreate -d service
+	$(COMPOSE) stop service
+
+# run alembic revision through the service pod
+# run processes as *local host user* inside the pod
+# preserve *local host user* files' uid inside the pod
+migrate-db: compose-for-migrate-db
+	podman run --rm -ti --user $(MY_ID) --uidmap=$(MY_ID):0:1 --uidmap=0:1:999 \
+	-e DEPLOYMENT=dev \
+	-e REDIS_SERVICE_HOST=redis \
+	-e POSTGRESQL_USER=packit \
+	-e POSTGRESQL_PASSWORD=secret-password \
+	-e POSTGRESQL_HOST=postgres \
+	-e POSTGRESQL_DATABASE=packit \
+	-v $(CURDIR)/alembic:/src/alembic:rw,z \
+	-v $(CURDIR)/packit_service:/usr/local/lib/python3.9/site-packages/packit_service:ro,z \
+	-v $(CURDIR)/secrets/packit/dev/packit-service.yaml:/home/packit/.config/packit-service.yaml:ro,z \
+	-v $(CURDIR)/secrets/packit/dev/fullchain.pem:/secrets/fullchain.pem:ro,z \
+	-v $(CURDIR)/secrets/packit/dev/privkey.pem:/secrets/privkey.pem:ro,z \
+	--network packit-service_default \
+	quay.io/packit/packit-service:dev alembic revision -m "$(CHANGE)" --autogenerate
+	$(COMPOSE) down
```

Comment:
I guess the purpose of this `$(COMPOSE) down` is to stop the container/pod started by podman in the previous step?
But the container/pod should exit/stop once the `alembic revision` finishes, so I don't see a point of this `$(COMPOSE) down`.